### PR TITLE
Static file service

### LINKED ISSUE #530: Allow returning file objects from a service
Currently, if you want to return a file from a service, you have to use a redirect or read the file yourself.  If `invoke_service` returned `cherrypy.lib.file_generator` instances directly, then you could return a file via `return cherrypy.lib.static.serve_file(path, content_type=mime_type)`, which would be cleaner than using a 302 or 307 redirect.  This should be a one line change, but then tests would need to be added.

### PR DESCRIPTION
**Pull request by ronichoudhury**
Closes #530

### CODE REVIEW

File: tests/staticfile.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import fixture
+
+
+@nose.with_setup(fixture.start_tangelo, fixture.stop_tangelo)
+def test_static_file():
+    r = requests.get(fixture.url("static_file"))
+    print r.text
+    assert r.ok
+    assert r.status_code == 200
+    assert r.text == "Infinite Diversity in Infinite Combinations\n"
```
Comment (manthey) [inline tests/staticfile.py]: We might as well test the mime type pass-though aspect of this.

For instance, you could add to the end of this file:
```
    assert "text/plain" in r.headers["Content-Type"]
    r = requests.get(fixture.url(
        "static_file?mime_type=application/octet-stream"))
    assert r.ok
    assert r.status_code == 200
    assert r.text == "Infinite Diversity in Infinite Combinations\n"
    assert "application/octet-stream" in r.headers["Content-Type"]
```
and alter `tests/web/static_file.py` to be
```
def run(**kwargs):
    return tangelo.file("../static_file.txt",
                        content_type=kwargs.get("mime_type", "text/plain"))
```

Comment (manthey) [inline tests/staticfile.py]: If you want, I can push these changes.

Comment (ronichoudhury) [inline tests/staticfile.py]: Sounds good, go for it.

**Comment by manthey** [comment]
Your changes LGTM.  If you like my changes, go ahead and merge.

**Pull request merged**